### PR TITLE
Delete file 

### PR DESCRIPTION
**Pull request by Mostafa-GamalEldin**

### CODE REVIEW

File: src/testFile.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 This file for new branch
+hello
```
Comment (KareemZafan) [inline src/testFile.txt]: Thanks you did a great job

Comment (KareemZafan) [reply src/testFile.txt]: Capatilize hello.

**Review by mohamedfarse7500 — APPROVED** [review]

File: src/testFile.txt (hunk 1 above)
Comment (Mostafa-GamalEldin) [reply src/testFile.txt]: I will apply

**Review by KareemZafan — APPROVED** [review]
Thank you

**Pull request merged**